docs/release-notes/2.31.0.md

```diff
@@ -0,0 +1,166 @@
+    
+    The dashboards can now be searched by panel title as well as dashboard title. Also, the overall performance has been enhanced as querying and sorting have become much faster.
+
+- **Default unified alerting**
+
+    Grafana Alerting is now on by default.
+
+## Vacuum monitoring dashboard
+
+!!! caution alert alert-warning "Important"
+    These experimental dashboards are subject to change. It is recommended to use these dashboards for testing purposes only.
+
+
+A new experimental dashboard has been introduced to overcome the difficulties related to vacuum by providing them with the correct and timely insights.
+
+This dashboard contains the following:
+
+- Dead tuples - Identifies the number of dead rows in each table even though the rows are physically removed from the table.
+
+- Last time vacuum ran - Tracks the last time a vacuum or autovacuum process successfully ran on each of your tables.
+
+- Number of rows modified since last analyze - The number of rows changed since the last time ANALYZE ran.
+
+- Manual vacuum events - Tracks the number of times a manual vacuum was run on each table. 
+
+- Table disk usage - Tracking the disk space used by each table is crucial as it enables you to gauge expected changes in the query performance over time—but it can also help you detect potential vacuuming-related issues.
+
+
+## Podman GA
+
+We are excited to announce the General Availability (GA) of Podman support for Deploying PMM 2.31.0. We had introduced it in 2.29.0 as a preview feature, but now we are production ready with this feature.
+
+
+## New Alerting
+
+Percona Alerting is a solution based on Grafana Alerting with the addition of simplicity of Templates usage.  
+This will enable users to use Alerting without deep knowledge of metrics details and consume the best knowledge from Percona Experts who created Alerting Templates. 
+
+
+
+### Simplied deployment with Database as a Service (DBaaS)
+
+In our constant endeavor and focus on an enhanced user experience, in PMM 2.31.0, we have simplified the deployment and configuration of DBaaS as follows:
+
+- With PMM 2.31.0, you can easily add a DB cluster from the newly created K8s cluster. All the DB cluster window fields are auto-populated with the values based on the existing K8s cluster. 
+
+- For PMM 2.31.0, while accessing DbaaS, if you have an existing Kubernetes cluster configured for DBaaS, you will be automatically redirected to the DB Cluster page. Otherwise, you would be redirected to the Kubernetes Cluster page.
+
+- We have added support for the latest (?) released Percona Operator for MongoDB (link to operator releases).
+
+
+
+### Operators support
+
+PMM now supports Percona Operator for MongoDB [1.12.0](https://docs.percona.com/percona-operator-for-mongodb/RN/Kubernetes-Operator-for-PSMONGODB-RN1.12.0.html).
+
+
+
+
+
+## New Features
+
+- [PMM-10089](https://jira.percona.com/browse/PMM-10089): We have upgraded Grafana to 9.1 in PMM 2.31.0.
```

Comment:
I think this item should be highlighted because we upgraded the major version of Grafana.

Reply:
@BupycHuk Its already a part of Release Highlights